src/server/document-controller.js

```diff
@@ -84,8 +80,8 @@ class DocumentController {
 
     const room = doc.roomId ? await this.roomService.getRoomById(doc.roomId) : null;
 
-    if (room?.access === ROOM_ACCESS.private && !isRoomOwnerOrMember({ room, userId: user._id })) {
-      throw new Forbidden();
+    if (!!room && !isRoomOwnerOrMember({ room, userId: user._id })) {
+      throw new Unauthorized();
```

Comment:
I think this was correct before. `Unauthorized` (401, bad naming for "unauthenticated") will simply redirect the user to the login page (and therefore into an endless loop), while we want `Forbidden` (403) :thinking:

Reply:
Ah so this isn't actually `Unauthorized` 😱 Would have been better than `Forbidden`.. Damn, ok, will revert all these places where I changed it.

Reply:
Reverted. Again what learned, thanks!

Reply:
Yeah, not optimal naming from the early days of the web :-) Thanks!